src/olympia/reviewers/templates/reviewers/dashboard.html

```diff
@@ -0,0 +1,21 @@
+{% extends "reviewers/base.html" %}
+
+{# Obsolete blocks that are not supposed to be displayed in the new reviewer UI #}
+{% block navbar %}{% endblock %}
+{% block pillnav %}{% endblock %}
+{% block header_search %}{% endblock %}
```

Comment:
long term are we dropping these from the base reviewer template?

Reply:
Yes